sass/checkout-layout-rtl.scss

```diff
@@ -0,0 +1,72 @@
+// ------------------------------------
+// IMPORT ABSTRACT
+// ------------------------------------
+@import
+	"abstract/functions/_all-early", // Should be imported before `_variables`
+	"abstract/variables-rtl",
+	"abstract/variables",
+	"abstract/functions/_all",
+	"abstract/mixins/_all";
+
+
+
+
+
+// ------------------------------------
+// IMPORT STYLES
+// ------------------------------------
+@import
+	"partials/collapsible-blocks",
+	"partials/checkout-content-area",
+	"partials/checkout-header",
+	"partials/checkout-progress-bar",
+	"partials/checkout-steps",
+	"partials/express-checkout",
+	"partials/checkout-login",
+	"partials/checkout-contact",
+	"partials/checkout-billing",
+	"partials/checkout-shipping",
+	"partials/checkout-shipping-methods",
+	"partials/checkout-payment-methods",
+	"partials/checkout-coupon-code",
+	"partials/checkout-place-order",
+	"partials/checkout-sidebar",
+	"partials/woocommerce-form-fields",
+	"partials/woocommerce-form-fields-select2",
+	"partials/order-summary",
+	"partials/expansible-sections";
+
+
+// ------------------------------------
+// IF NOT EXITS WITH VARIABLES
+// ------------------------------------
+body.woocommerce-checkout form.woocommerce-checkout .fc-wrapper .fc-step__substep-fields,
+div.woocommerce .fc-expansible-form-section .expansible-section__toggle-plus{
+	text-align: $checkout-rtl-alignment-left;
+}
+
+.has-checkout-layout--multi-step.woocommerce-checkout form.woocommerce-checkout .fc-wrapper [data-step-complete] .fc-step__substep .fc-step__substep-title:before {
+    right: 0px;
+}
+
+.has-checkout-layout--multi-step.woocommerce-checkout form.woocommerce-checkout .fc-wrapper [data-step-complete] .fc-step__substep .fc-step__substep-title {
+    float: right;
+    padding: 0 28px 0 0px;
+}
+
+form.woocommerce-checkout .fc-wrapper #order_review table.woocommerce-checkout-review-order-table .product-quantity {
+    right: 65px;
+    left: auto;
+    transform: translateX(100%) translateY(-25%);
+}
+.has-checkout-layout--multi-step .fc-step__substep-text {
+    padding-left: 0;
+    padding-right: 80px;
+    width: calc( 100% - 230px );
+}
+
+div.woocommerce form .form-row.woocommerce-validated:not([class*=woocommerce-invalid]):not(.fc-no-validation-icon) .select2-selection:after,
+div.woocommerce form .form-row.woocommerce-validated:not([class*=woocommerce-invalid]):not(.fc-no-validation-icon):not(.fc-select2-field) .woocommerce-input-wrapper:after {
+    left: 20px;
+    right: auto !important;
```

Comment:
@alamgircsebd Replace `px` with `remsize()`, and use new alignment variables for this code.